src/app-layer-htp.c

```diff
@@ -179,6 +179,11 @@ SCEnumCharMap http_decoder_event_table[ ] = {
     { "REQUEST_LINE_INCOMPLETE",
         HTTP_DECODER_EVENT_REQUEST_LINE_INCOMPLETE},
 
+    { "LZMA_SUSPICIOUS_DICT_RATIO",
```

Comment:
Should this be `DIST`?

Reply:
No it was about a dictionary size. But this is outdated code, in the newer this isn't present.